backend/src/main/java/com/google/sps/data/Participant.java

```diff
@@ -26,6 +28,7 @@ public Participant(
       long endTimeAvailable,
       int duration,
       long matchId,
+      MatchStatus matchStatus,
```

Comment:
A potential cleanup for a future PR could be to make this constructor private and instead use public factory methods to create participants. Some of the code that calls this now has to specify fields that don't make sense for their particular case. For example, creating a new unmatched participant requires the caller to explicitly include the MatchStatus and a zeroed out match ID. Not only does this lead to more LOC, but it also leaks a small detail that 0 is this class' default value for matchId. 

A factory method might look like

```
public Participant createUnmatchedParticipant(String username, long startTimeAvailable, long endTimeAvailable, int duration, long timestamp) {
  return new Participant(username, startTimeAvailable, endTimeAvailable, /* matchId= */ 0, MatchStatus.UNMATCHED, timestamp)
}
```

which requires two fewer arguments because an unmatched participant will always have the unmatched enum value and zeroed match ID. 

Not for this PR, but something to think about.

Reply:
Ah interesting. I'll add this as an issue